src/ctia/http/routes/common.clj

```diff
@@ -398,6 +401,28 @@
   (st/merge PagingParams
             RelationshipFieldsParam))
 
+;; scratchpad
+
+(s/defschema ScratchpadFieldsParam
+  {(s/optional-key :fields) [scratchpad-sort-fields]})
+
+(s/defschema ScratchpadSearchParams
+  (st/merge
+   PagingParams
+   BaseEntityFilterParams
+   SourcableEntityFilterParams
+   ScratchpadFieldsParam
+   {:query s/Str
+    (s/optional-key :texts.text) s/Str
```

Comment:
I know that certainly "works™" right now. But this is "dangerous" from:

https://clojure.org/reference/reader

> Keywords - Keywords are like symbols, except:
> ... they cannot contain '.' or name classes.

So the solution might be to use strings instead of keywords for query if possible and easy to do. Certainly something to point in another technical issue. I imagine the problem is the same for all other entities.